.dotnet/src/Custom/Chat/ChatTokenUsage.cs

```diff
@@ -7,12 +7,28 @@ namespace OpenAI.Chat;
 public partial class ChatTokenUsage
 {
     // CUSTOM: Renamed.
-    /// <summary> Number of tokens in the generated completion. </summary>
+    /// <summary>
+    /// The combined number of output tokens consumed by the model.
+    /// </summary>
+    /// <remarks>
+    /// When using a model that supports <see cref="ReasoningTokens"/> such as <c>o1-mini</c>, this value represents
+    /// the sum of those reasoning tokens and conventional, displayed output tokens.
+    /// </remarks>
     [CodeGenMember("CompletionTokens")]
     public int OutputTokens { get; }
```

Comment:
Maybe for a separate PR, but I wonder if these should be called "OutputTokenCount", "InputTokenCount", etc.